pages/index.tsx

```diff
@@ -29,14 +29,14 @@ const Home: React.FC<HomePageProps> = (props: HomePageProps) => {
   return (
     <Page className="flex align-items-center">
       <div className="flex-1 width-full">
-        <section className="padding-top-4xl padding-bottom-xl">
+        <section className="padding-top-4xl padding-bottom-l">
           <PageContainer>
-            <div className="padding-left-l margin-bottom-l m:margin-bottom-4xl">
+            <div className="padding-left-l l:padding-left-none margin-bottom-l s:margin-bottom-4xl">
               <div className="width-full flex flex-row justify-content-between align-items-center">
                 <h2 className="heading-2">Vaccination rates</h2>
                 {hasMounted && <DarkModeToggle className="flex-0" />}
               </div>
-              <h1 className="heading-1">Aotearoa</h1>
+              <h1 className="heading-1 margin-bottom-xl">Aotearoa</h1>
```

Comment:
Can we extend the spacing scale, rather than using additive margin to create enough space? IMO all the margin should be on the parent div here

Reply:
Yep, good call!